.gitignore

```diff
@@ -18,3 +18,6 @@ doc/_static/thumbs/*
 doc/tutorials_apps/*
 !doc/_static/code.png
 doc/code/api/*
+Pipfile*
+.vscode/
+.ipynb_checkpoints/*
```

Comment:
was there any specific reason for adding these?

Reply:
I didn't want to accidentally commit files by mistake, so with that they don't even show up